backend/src/bundles/users/user.service.ts

```diff
@@ -43,14 +45,25 @@ class UserService implements Service {
         };
     }
 
-    public async create(payload: UserSignUpRequestDto): Promise<UserEntity> {
-        return this.userRepository.create(
+    public async create(
+        payload: UserSignUpRequestDto,
+    ): Promise<UserFindResponseDto> {
+        const { PASSWORD_SALT_ROUNDS } = config.ENV.CRYPT;
```

Comment:
save PASSWORD_SALT_ROUNDS as private field for UserService class and pass it in constructor instead of using from import

Reply:
@MaksymGavatuk done